docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
     env_file: d4s2.env
     depends_on:
       - db
+  ui:
+    build: https://github.com/Duke-GCB/datadelivery-ui.git#ember-jwt
```

Comment:
Need to remove the "#ember-jwt" part above once https://github.com/Duke-GCB/datadelivery-ui/pull/15 is merged.